dev-python/flask-peewee/flask-peewee-0.6.7-r1.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
```

Comment:
That doesn't look like 'fixing deps' to me. Really belongs in a separate commit (to the same revision).